packages/protocol-contracts/test/ZetaTokenConsumer.spec.ts

```diff
@@ -267,7 +267,7 @@ describe("ZetaTokenConsumer tests", () => {
       const result = await tx2.wait();
 
       const eventNames = parseZetaConsumerLog(result.logs);
-      expect(eventNames.filter((e) => e === "ZetaExchangedForToken")).to.have.lengthOf(1);
+      expect(eventNames.filter(e => e === "ZetaExchangedForToken")).to.have.lengthOf(1);
```

Comment:
According to our linter config this should be (e), idk what happened to the autofix 🤔.

Reply:
🤬